fixture.js

```diff
@@ -1,8 +1,11 @@
 // This is not part of the project, just a simple program that keeps the
 // event loop open. Run as `node -r ./lib/index.js fixture.js`
+/* eslint-disable no-unused-vars */
 'use strict';
 const fs = require('fs');
 const http = require('http');
+const natives = process.binding('natives');
```

Comment:
do you need to assign it?  Can you just `process.binding('natives');` and be done